firedrake/slate/slate.py

```diff
@@ -567,7 +568,7 @@ class Action(TensorOp):
     :arg function: a :class:`firedrake.Function` object.
     """
 
-    prec = 2
+    prec = 3
```

Comment:
I probably missed something earlier, but what are these precedences actually used for?